.github/s390x-ci/self-hosted-builder/fs/usr/bin/actions-runner

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Nit (I've seen system where bash is in usr/bin rather than bin, but I've yet to see a Linux where env is not in usr/bin)
```suggestion
#!/usr/bin/env bash
```

Reply:
Thanks, I'll update this